### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by scarroll32** [lloyd/JSONSelect#69]
Looks like an amazing piece of work. Has it been forked or continued elsewhere?

**Comment by seanfcarroll** [comment]
Looks like an amazing piece of work. Has it been forked or continued elsewhere?

**Comment by ghost** [comment]
I have to agree - this looks amazing. I have been looking for a standardized query language for JSON and there is _nothing_ out there. Any chance this project could be revived?

**Comment by chrisberkhout** [comment]
Looks like there have been only small tweaks since 2011. But maybe it's just working? I was pretty happy with it back then.

For a general purpose querying of JSON I'd recommend taking a look at [`jq`](https://stedolan.github.io/jq/) and [`ruby-jq`](https://github.com/winebarrel/ruby-jq).

Slightly off topic: I recently needed to diff JSON files and found [json-delta](http://json-delta.readthedocs.org/en/latest/) to be the best option.

**Comment by mindplay-dk** [comment]
I have to say, this makes way more sense and seems much more natural and intuitive than any other JSON query language I've looked at.

I too would like to know what is the state of this project?

Has it been implemented anywhere else?

**Comment by trvrm** [comment]
There's a python implementation that's pretty good.  (https://github.com/danvk/pyjsonselect).  I use both the JS and the Python version regularly.
